astropy/cosmology/core.py

```diff
@@ -395,7 +400,7 @@ def __repr__(self):
         if self.name is not None:
             namelead += f'name="{self.name}", '
         # nicely formatted parameters
-        fmtps = (f"{k}={getattr(self, k)}" for k in self.__parameters__)
+        fmtps = (f"{k}={getattr(self, k)}" for k in self._parameters_)
```

Comment:
Why not just use the values in `self._parameters`? I.e., iterate over `k, v in self._parameters.items()`

Reply:
SGTM.

Reply:
Done!